bsc.typechecker/higherrank/higherrank.exp

```diff
@@ -0,0 +1,9 @@
+
+# Tests for higher-rank struct fields
+
+test_c_veri HigherRank
+test_c_veri Undefined
+test_c_veri DeepSeqCond
+
+# Fails in elab
+compile_pass Uninitialized.bs
```

Comment:
It might help to be less brief in the comment.  Maybe say, "Use of an uninitialized value results in a expected error".

It might also be worth actually trying to compile the file for Verilog and confirming that it generates the expected error.  (For example, by using `compile_verilog_fail_error` and looking specifically for `G0028`.)  If BSC gets as far as generating the `G0028` error, then we also know that typecheck succeeded, so this would subsume the `compile_pass` check.